course/grades.py

```diff
@@ -295,43 +308,195 @@ def grade_key(entry):
         })
 
 
+class ExportGradeBookForm(StyledForm):
+    def __init__(self, *args, **kwargs):
+        # type: (*Any, **Any) -> None  # noqa
+        super(ExportGradeBookForm, self).__init__(*args, **kwargs)
+
+        self.fields["user_info_fields"] = forms.ChoiceField(
+            choices=(csv_handler.export_csv_fields_options),
+            initial=csv_handler.export_csv_fields_options[0],
+            label=_("User Fields to Export"),
+        )
+
+        self.fields["exclude_instructors"] = forms.BooleanField(
+            required=False,
+            initial=True,
+            label=_("Exclude Grades of Instructors"),
+        )
+
+        self.fields["exclude_tas"] = forms.BooleanField(
+            required=False,
+            initial=True,
+            label=_("Exclude Grades of Teaching assistants"),
+        )
+
+        self.fields["zero_for_state_none"] = forms.BooleanField(
+            required=False,
+            initial=False,
+            label=_("Use 0 point for grade state `NONE`, which possibly "
+                    "means the opptunity had not been graded."),
+        )
+
+        self.fields["zero_for_graded_none"] = forms.BooleanField(
+            required=False,
+            initial=True,
+            label=_("Use 0 for graded opptunity which has `NONE` points"),
+        )
+
+        self.fields["maximum_points"] = forms.FloatField(
+            required=True,
+            min_value=0,
+            max_value=100 * MAX_EXTRA_CREDIT_FACTOR,
+            initial=100,
+            label=_("Maximum allowed points"),
+        )
+
+        self.fields["minimum_points"] = forms.FloatField(
+            required=True,
+            initial=0,
+            min_value=0,
+            max_value=100 * MAX_EXTRA_CREDIT_FACTOR,
+            label=_("Minimum points"),
+        )
+
+        self.fields["round_digits"] = forms.IntegerField(
+            required=True,
+            min_value=0,
+            initial=0,
+            max_value=2,
+            label=_("Round Digits"),
```

Comment:
Help text.

Reply:
Or maybe change label to "Round point counts to N digits" and "Round percentages to N digits". Also, why max out at 2?

Reply:
I previously set that to 4, and I found (in tests) the value are all rounded to 2 digits, had thought that's preset behavior. Since you commented on this,  I'll check that.